### PR TITLE
Add '--include-full-xr' flag to the render subcommand

### PR DESCRIPTION
**Pull request by ezgidemirel**
Documents the new render flag `--include-full-xr`.

### CODE REVIEW

File: content/master/cli/command-reference.md [hunk 1]
```diff
@@ -388,6 +388,7 @@ spec:
 |              | `--context-files=<key>=<file>,<key>=<file>`    | A comma separated list of files to load for function "contexts." |
 |              | `--context-values=<key>=<value>,<key>=<value>` | A comma separated list of key-value pairs to load for function "contexts."                                                    |
 | `-r`         | `--include-function-results`          | Include the "results" or events from the function.   |
+| `-x`         | `--include-full-xr`          | Include a direct copy of the input XR's spec and metadata fields in the rendered output.   |
```
Comment (plumbis) [inline content/master/cli/command-reference.md:391]:
```suggestion
| `-x`         | `--include-full-xr`          | Include a copy of the input Composite Resource spec and metadata fields in the rendered output.   |
```

File: content/master/cli/command-reference.md [hunk 2]
```diff
@@ -409,6 +410,14 @@ If a function produces Kubernetes events with statuses use the
 `--include-function-results` to print them along with the managed resource 
 outputs. 
 
+#### Include the full composite resource in the output
+
+Composition functions can only modify the `status` field of a composite 
+resource. Therefore, the `crossplane beta render` command only prints the
+`status` field with `metadata.name`. With the `--include-full-xr` flag 
+the command prints the full composite resource, including the `spec` 
+and `metadata` fields.
```
Comment (plumbis) [inline content/master/cli/command-reference.md:419]:
```suggestion
#### Include the composite resource 

Composition functions can only change the `status` field of a composite 
resource. By default, the `crossplane beta render` command only prints the
`status` field with `metadata.name`.  

Use `--include-full-xr` to print the full composite resource, 
including the `spec` and `metadata` fields.
```
Suggestions to appease our lord and savior Vale 😆

**Review by plumbis — CHANGES_REQUESTED** [review]

**Comment by plumbis** [comment]
Resolves #683

**Review by phisco — APPROVED** [review]

**Comment by phisco** [comment]
obviously committing from the GUI breaks the DCO 😭 I can't force push on this PR, so I'll just open another PR with the same changes

**Pull request closed without merging**